### ISSUE TITLE
About "Static" tag to the website

### ISSUE THREAD

**Issue by chaitanyachavali** [MozPhoenix-Community/siNUsoid-17#2]
Does the website intend to be static? 

- If **yes**, any strong reasons to be so?
- If **no**, what are the features that are to be added?

**Comment by ghost** [comment]
NO, the website doesn't mean to be a static one, as far as I think.
There are several features, that I will be working on, after the overall design is completed.
Some of them are:-

* Dynamic addition and updation of events (But I don't think this is necessary)
* Live updates from events through the android app given to the organizing team (including the winner announcements)
* The individual event registrations and the overall fest attendance will be handled from the website only. So, that's where the back-end comes into the picture.
* Also, through android, I plan to integrate the entry token that will be generated through the ticketing system with QR crypto and databases to maintain the attendance, as mentioned in the previous point.
* More points, coming soon :P
